vignettes/more-recipes.Rmd

```diff
@@ -0,0 +1,277 @@
+---
+title: Apply other steps and models from recipes and parsnip packages
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Apply other steps and models from recipes and parsnip packages}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+The `epipredict` package utilizes the `tidymodels` framework, namely 
```

Comment:
Add section: Not doing it the Epipredict for lagging and leading. Show demo of doing it without step_lag/lead() and how it results in not using information from the most x-lagged time points. whether leading or lagging, they will be off by one time-period.